packages/interactor/test/definitions/text-field.test.ts

```diff
@@ -16,13 +16,30 @@ describe('@bigtest/interactor', () => {
 
     it('finds `input[type=text] tags by label', async () => {
       dom(`
-        <label type="text" for="name-field">Name</label><input id="name-field"/>
+        <label for="name-field">Name</label><input type="text" id="name-field"/>
       `);
 
       await expect(TextField('Name').exists()).resolves.toBeUndefined();
       await expect(TextField('Does not Exist').exists()).rejects.toHaveProperty('name', 'NoSuchElementError');
     });
 
+    it('finds inputs with custom type', async () => {
+      dom(`
+        <label for="name-field">Name</label><input type="monkey" id="name-field"/>
+      `);
+
+      await expect(TextField('Name').exists()).resolves.toBeUndefined();
+      await expect(TextField('Does not Exist').exists()).rejects.toHaveProperty('name', 'NoSuchElementError');
+    });
+
+    it('does not find non text field inputs', async () => {
+      dom(`
+        <label for="name-field">Name</label><input type="range" id="name-field"/>
+      `);
+
+      await expect(TextField('Name').exists()).rejects.toHaveProperty('name', 'NoSuchElementError');
+    });
+
```

Comment:
@cowboyd here's a test which checks that it does not match non-text-fields. It doesn't check button in particular, but enumerating all of them seemed a bit excessive to me.